keyboards/osx/mi-t-k0-osx.xml

```diff
@@ -3,7 +3,7 @@
 <keyboard locale="mi-t-k0-osx">
 	<version platform="10.9" number="$Revision$"/>
 	<names>
-		<name value="Maori"/>
+		<name value="Māori"/>
```

Comment:
Retract, probably want to leave the keyboard ones alone, since they are reflecting what the source data had.